src/deadline/nuke_submitter/assets.py

```diff
@@ -81,17 +81,41 @@ def get_scene_asset_references() -> AssetReferences:
             for filename in get_node_filenames(node):
                 asset_references.output_directories.add(dirname(filename))
 
-    # if using a custom OCIO config, add the config file and associated search directories
-    if nuke_ocio.is_custom_config_enabled():
-        ocio_config_path = nuke_ocio.get_custom_config_path()
-        ocio_config_search_paths = nuke_ocio.get_config_absolute_search_paths(ocio_config_path)
+    if nuke_ocio.is_OCIO_enabled():
+        # Determine and add the config file and associated search directories
+        ocio_config_path = get_ocio_config_path()
+        # Add the references
+        if ocio_config_path is not None:
```

Comment:
Is the ocio config path relative and not absolute? Are there any issues with path mapping or anything like that?

Reply:
ocio config path is in absolute. `get_ocio_config_path()` will return absolute path for stock and custom configurations. When testing, it didn't work properly with relative path.